### ISSUE TITLE
Improvement over git log?

### ISSUE THREAD

**Issue by jhermann** [karandesai-96/yolog#2]
You _do_ know about…

```
git log --all --color --graph --pretty=format:'%Cred%h%Creset -%C(yellow)%d%Creset %s %Cgreen(%cr)%C(bold blue) <%an>%Creset' --abbrev-commit
```

:question: 

Which [can be aliased](https://github.com/jhermann/ruby-slippers/blob/master/home/.gitconfig#L20) to make it reasonbly usable. So the README should probably tell us what's better.

**Comment by karandesai-96** [comment]
Yes, I do know about this. I initially went with a similar approach of using alias. But then I worked on a few repos locally where I am the author of all commits, so I can strip off name column. At someplaces I like to view emails of author as well. I know per-repository gitconfigs can exist, but personally I cluttered up using multiple aliases and overriding one over another and remembering which one should be turned on when. I decided a small wrapper (this package is < 5 kb in size) would be more efficient. Version 0.2.1 is just a minimal functional version, so I'll leave this issue open and I hope that the better things than `git log` would be visible more in subsequent versions. 

For the `alias` you've shown:
Mine displays results in columns, along with auto fitting them ( `%(<N>)` by git allows padding but I am not aware of auto fitting ). I won't say it is **better** than yours, it is just different - as different people would like it presented in different manners :smile: Well I am planning to solve this very thing in future version :- more customization as per choice of user / repository.

Thanks for pointing this out, I'll notify here when newer version is rolled out and README is updated.
